nav2_regulated_pure_pursuit_controller/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 3.5)
 project(nav2_regulated_pure_pursuit_controller)
 
 find_package(ament_cmake REQUIRED)
+find_package(nav2_amcl REQUIRED)
```

Comment:
TODO Me: remove AMCL dep and use `angles`

Reply:
oh yeah forgot about than one and your eyes when you saw that we are still using the good old amcl angles utils